src/styles/card.css

```diff
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  height: 406px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  display: flex;
+  padding: 0 3px;
+}
+
+.image {
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  color: var(--main-color);
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  padding-top: 40px;
+}
+
+.card__product-code {
+  display: flex;
+  color: var(--second-color);
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  padding-top: 4px;
```

Comment:
I think it's better to use margin-bottom instead of padding-top to follow native's browser rendering process (from up to bottom)